File: Python/002_Program/program.py
#!/usr/bin/env python3
# File Writer - Write text to file
def main():
    print("=== File Writer ===")
    filepath = input("Enter file path: ")
    print("Enter text (type 'END' to finish):")
    
    lines = []
    while True:
        line = input()
        if line == "END":
            break
        lines.append(line)
    
    with open(filepath, 'w') as f:
        f.write('\n'.join(lines))
    
    print("File saved successfully!")

if __name__ == "__main__":
    main()

File: CSharp/002_Program/Program.cs
// File Writer
using System;
using System.IO;
class Program {
    static void Main() {
        Console.Write("Enter file path: ");
        string path = Console.ReadLine();
        Console.WriteLine("Enter text (type END to finish):");
        using (StreamWriter sw = new StreamWriter(path)) {
            string line;
            while ((line = Console.ReadLine()) != "END") sw.WriteLine(line);
        }
        Console.WriteLine("File saved!");
    }
}